### DISCUSSION TITLE
Medusa v2.0 seeding external database?

### DISCUSSION THREAD

**Discussion by Shameel123** [medusajs/medusa#8423] [Q&A]
Hello,

I am in a bit of a pinch here. I have been able to successfully set up medusa v2.0 on local but now I am trying to point the backend to external db hosted on cloud platform.

I have successfully run migration for it but I am getting problem in how to seed the db.
I found this:
```
npx @medusajs/medusa-cli@latest seed -f ./data/seed.json
```

For v2, there seems to be `seed.ts` file.

It is not properly documented how to use this file to seed external database.

Can someone help in providing a proper guideline for how to do that?

Thanks in advance :)

**Comment by Shameel123** [comment]
Without seeding and just running the migrations is giving me this error currently:

> Error running script
> Error: Loaders for module PaymentModuleService failed: self-signed certificate in certificate chain

Not sure if it is related to Medusa v2 or with how the Postgresql is setup on cloud at this stage.
Just putting it here for now to see if someone knows how to get past this :)

**Reply by docloulou** [reply]
try `npx medusa exec ./src/scripts/seed.ts`

**Reply by Shameel123** [reply]
Yes, I had t ried it. Getting the same error as mentioned:

> error:   Error running script
> Error: Loaders for module PaymentModuleService failed: self-signed certificate in certificate chain
